Raspi/Test/mqtt_wireless_log.py

```diff
@@ -70,17 +72,29 @@ def on_message(client, userdata, msg):
 
 def start_recording(module_id_str):
     """ Start recording for a specific module """
-    # Clear the old temporary files in the folder, just in case the script
-    # exited early and they were not cleared properly
-    remove_files(find_temp_csvs(module_id_str))
+    # The old temporary files in the folder will not be removed just incase
+    # they contain just in case they need to be recovered.
 
     # Save the state of recording and the output filename to global dicts
     is_recording[module_id_str] = True
     module_start_time[module_id_str] = datetime.now()
 
-    # Generate filename from the current date and time in the Australian format
-    output_filename[module_id_str] = module_start_time[module_id_str].strftime(
-        module_id_str + "_D%d-%m-%d_T%H-%M-%S" + ".csv")
+    # Generate filename from the last log number + 1
+    max_file_id = 0
```

Comment:
This section is probably a great candidate for unit testing. Code that could be easy to screw up in the future, and also has scary consequences (loss of data).

You could separate this into a function that accepts a list of file names, and returns a correct new filename. Then it can be called here and tested from elsewhere

Reply:
Next PR, should be a unit test solely to protect edge cases are not happening.